nncf/common/graph/graph.py

```diff
@@ -331,7 +331,24 @@ def get_input_edges(self, node: NNCFNode) -> List[NNCFGraphEdge]:
         :return: List of input edges for the node sorted by input port ID.
         """
         input_nodes = self.get_previous_nodes(node)
-        edges = [self.get_edge(from_node, node) for from_node in input_nodes]
+        edges = []
+        for from_node in input_nodes:
+            edge = self.get_edge(from_node, node)
+            if not edge.parallel_input_port_ids:
+                edges.append(edge)
+                continue
+            for input_port_id in edge.parallel_input_port_ids:
```

Comment:
```suggestion
            if len(edge.parallel_input_port_ids) == 0:
                edges.append(edge)
            for input_port_id in edge.parallel_input_port_ids:
```

Minor, according to my taste :)

Reply:
Please check

Reply:
According to [PEP8](https://peps.python.org/pep-0008/#programming-recommendations) it's better to use `not` instead of `len(...) == 0`